package.json

```diff
@@ -26,20 +26,21 @@
   },
   "homepage": "https://github.com/geostyler/geostyler-cql-parser#readme",
   "scripts": {
+    "generate": "jison -m commonjs ./src/cql.jison -o ./src/cql-parser.js",
     "build:browser": "webpack --config browser-build.config.js",
     "build": "tsc -p tsconfig.json && npm run build:browser",
     "lint": "eslint -c .eslintrc.js --ext ts,spec.ts src/",
     "typecheck": "tsc --noEmit --project tsconfig.json",
-    "prebuild": "npm run test",
+    "prebuild": "npm run generate && npm run test",
```

Comment:
does this run `npm run generate` twice, since it is already included in `pretest`?

Reply:
True, I removed it here